amr-wind/wind_energy/ABLWallFunction.cpp

```diff
@@ -292,12 +341,17 @@ void ABLTempWallFunc::wall_model(
     const auto& density = repo.get_field("density", rho_state);
     const auto& alpha = repo.get_field("temperature_mueff");
     const int nlevels = repo.num_active_levels();
-
+    const bool is_surf_temp = repo.field_exists("surf_temp");
+    const auto* m_surf_temp =
+        is_surf_temp ? &repo.get_field("surf_temp") : nullptr;
+    const bool is_multi = repo.field_exists("lowerz0");
```

Comment:
would prefer a better name than `is_multi`

Reply:
Nothing came to mind.

Reply:
`has_variable_roughness` or `has_variable_surface` ?